Python/ast_unparse.c

```diff
@@ -67,11 +67,11 @@ replace_inf_value(PyObject *repr)
         return NULL;
     }
     PyObject *inf_value = PyUnicode_FromFormat("1e%d", 1 + DBL_MAX_10_EXP);
+    Py_DECREF(inf_str);
```

Comment:
No. That's wrong. I only suggest to add Py_DECREF(inf_str); if inf_value == NULL.

Reply:
right